bin/activate-hermit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This file must be used with "source bin/activate-hermit" from bash or zsh.
```

Comment:
so I run this to set up my environment after pulling this change?

Reply:
the home README.md should be updated to say how you get dev env set up with hermit

Reply:
https://github.com/cashapp/backfila/pull/291